web/src/SelectThemeBox.js

```diff
@@ -18,61 +18,75 @@ import {Dropdown} from "antd";
 import "./App.less";
 import i18next from "i18next";
 
-function themeIcon(themeKey) {
-  return <img width={24} alt={themeKey} src={getLogoURL(themeKey)} />;
+export const Themes = [
+  {label: "Default theme" || i18next.t("theme:Default theme"), key: "default", icon: `${Setting.StaticBaseUrl}/img/light.svg`},
```

Comment:
It is for script to generate the i18 word key. Because the usage is `i18.next.t( theme:${theme.label}`). Casdoor generate script can not recognize it.

Reply:
Why not directly use `i18next.t("theme:Default theme")`?